### PR TITLE
feat(hosting): add retention advices

### PR DESCRIPTION
**Pull request by ghyenne**
ref MANAGER-8365

Signed-off-by: Guillaume Hyenne <guillaume.hyenne@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `release/2022-w09`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-8365 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated (n/a)~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

I added retention advices in hosting section
ref: MANAGER-8365.

### :flags: Translations
- [x] TRANS-41697 (2api)

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html [hunk 1]
```diff
@@ -368,14 +368,12 @@
                         data-term="{{:: 'hosting_dashboard_service_cdn' | translate }}"
                     >
                         <oui-tile-description>
-                            <span
-                                data-ng-if=""
-                            >
+                            <span data-ng-if="">
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:371]: **nitpick(non-blocking):** Following expression will always be evaluated as falsy, so we can safely get rid of this `<span>` element.

**Review by antleblanc — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (radireddy) [reply packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:371]: Looks like its the old, existing code.

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (ghyenne) [reply packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:371]: Is existing code, not add by me. Lint has modified structure. This span has the content. It's a temporary deletion? Should I delete it?

**Review by varun257 — DISMISSED** [review]

File: packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html (hunk 1 above)
Comment (antleblanc) [reply packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:371]: Yes please! We can safely remove this `<span>` element and its translation here:
https://github.com/ovh/manager/blob/5251e0ec44723fce706bd5283d759de0c59a5899/packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json#L151

Comment (JeremyDec) [reply packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html:371]: In all cases, as @antleblanc mentioned it, the `ng-if` will always returns false => the chlid span has its own condition, so yes you can remove it

**Review by JeremyDec — DISMISSED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by radireddy — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Pull request merged**